pkg/reconciler/openshift/common/transformer.go

```diff
@@ -59,10 +59,6 @@ func UpdateDeployments(prefix string, replaceImg map[string]string, skipImages [
 
 			// Prefix Images in Args if there
 			for i := 0; i < len(c.Args); i++ {
-				if valueInArr(skipImages, c.Args[i]) {
-					i++
-					continue
-				}
```

Comment:
hi, im just curious. 
why does this patch remove this.

Reply:
this code was specific to skip an image 
func does 2 things
- add prefix to images 
- replace image with some other in some cases
There was 3rd where we use to skip the image but this code was specific to the 3rd case which only for gsutil image
and as we are removing external ref from payload in future too we will be doing that 
so there won't a case to skip an image.